.github/workflows/test.yml

```diff
@@ -29,8 +29,8 @@ jobs:
         run: python3 -m pip install --upgrade pip setuptools wheel
       - name: Install mido in dev mode
         run: python3 -m pip install --quiet .[lint-code]
-      - name: Lint code with flake
-        run: flake8
+      - name: Lint code with ruff
+        run: ruff .
```

Comment:
I half-disagree: it might be useful to see possible warnings, so unless you vehemently disagree, I'll skip `--quiet`.

I'll add the explicit `check` and also add the `RUFF_OUTPUT_FORMAT: github` envvar so checks will surface as GitHub check warnings.

Reply:
I just based my assessment on the text output diff and found the --quiet a bit less verbose yet not silent.
I'll trust your judgement on this.
Adding github formatting would be a nice touch, yes!